### ISSUE TITLE
Lack of SI Prefixes listing in documentation

### ISSUE THREAD

**Issue by quctex** [yt-project/unyt#268]
The [documentation](https://unyt.readthedocs.io) contains sections "Listing of Units" and "Listing of Physical Constants". It would be great to add another section "Listing of Unit Prefixes" or "Listing of SI Prefixes" there.

Current workaround is to find them in source code [here](https://github.com/yt-project/unyt/blob/master/unyt/_unit_lookup_table.py#L380) or try to guess. This can be difficult for the average user.

While these prefixes are generic and easy to guess, this simple addition will make the documentation more complete and beginner friendly. 

In addition, there is an unobvious thing - Unicode characters for the "micro" prefix, and it would be good to reflect this in the documentation.

**Comment by jzuhone** [comment]
@quctex I agree--I can issue a PR for this or please feel free to do it also!
